src/Raven.Server/Documents/Patch/PatchDocumentCommand.cs

```diff
@@ -1,16 +1,31 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using Esprima.Ast;
 using Jint.Native;
 using Raven.Client.Documents.Operations;
 using Raven.Client.Exceptions;
+using Raven.Client.Extensions;
 using Raven.Server.ServerWide.Context;
 using Sparrow.Json;
+using Sparrow.Json.Parsing;
 
 namespace Raven.Server.Documents.Patch
 {
-    public class PatchDocumentCommand : TransactionOperationsMerger.MergedTransactionCommand, IDisposable
+    public class PatchDocumentCommand : TransactionOperationsMerger.MergedTransactionCommand, IDisposable, TransactionOperationsMerger.IRecordableCommand
     {
+        private const string IdKey = "Id";
```

Comment:
No, use `nameof`, instead

Reply:
It does not matter whether the field is private?

Reply:
done